include/sound/sof/pm.h

```diff
@@ -45,4 +45,20 @@ struct sof_ipc_pm_core_config {
 	uint32_t enable_mask;
 } __packed;
 
+struct sof_ipc_pm_gate {
+	struct sof_ipc_cmd_hdr hdr;
+	uint32_t flags;
+} __packed;
+
+#define SOF_PM_PG_RSVD		BIT(0)
+
+/** \brief Indicates whether streaming is active */
+#define SOF_PM_PG_STREAMING	BIT(1)
+
+/** \brief Prevent power gating (0 - deep power state transitions allowed) */
+#define SOF_PM_PPG		BIT(2)
+
+/** \brief Prevent clock gating (0 - cg allowed, 1 - DSP clock always on) */
+#define SOF_PM_PCG		BIT(3)
```

Comment:
here we go: double negatives....

If you 'gate' and send a 'prevent' flag then it's confusing what is actually done.

Can we please use 'active' flags, where it's explicit what the ask is.

Reply:
This is what @mmaka1 defined in his PR https://github.com/thesofproject/sof/pull/1729, I think the reason is that those bits defined in register is 'prevent' xx gating. 

@mmaka1 can you comment, I am fine if you do change here, finalize in your PR and here I will follow.